src/Concept.java

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
I suspect that this doesn't compile. It might in newer versions, but it seems like the class declaration is missing.